go/tasks/plugins/array/awsbatch/transformer.go

```diff
@@ -144,7 +144,7 @@ func getEnvVarsForTask(ctx context.Context, execID pluginCore.TaskExecutionID, c
 			Value: val,
 		})
 	}
-
+	finalEnvVars = append(finalEnvVars, v1.EnvVar{Name: failOnError, Value: "true"})
```

Comment:
Can we instead use something like:
```
m[failOnError] = "true"
```
on like line 139? Then it will be populated in the `finalEnvVars` as part of the loop and the array with be right-sized rather than reallocating capacity because it's set as `len(m)`. Also, it stops from double-setting the env var.

Reply:
done, thanks